pkg/sql/conn_executor.go

```diff
@@ -3687,3 +3759,56 @@ func init() {
 		return planGistFromCtx(ctx)
 	})
 }
+
+type jobsInfo struct {
+	jobsCreated           jobsCollection
+	uniqueJobsToCreate    map[descpb.ID]*jobs.Record
```

Comment:
nit: name this `uniqueToCreate` and the other one `nonUniqueToCreate` and use comments to explain what makes them different.

Reply:
done.